packages/application-components/src/components/dialogs/info-dialog/info-dialog.mdx

```diff
@@ -0,0 +1,65 @@
+---
+id: components-info-dialog
+title: Info Dialog - Components
+permalink: /components/info-dialog
+---
+
+# InfoDialog
+
+import { InfoDialog } from '@commercetools-frontend/application-components';
```

Comment:
This is the interactive part. The code below is basically what we had in the VRT test.